packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts

```diff
@@ -40,6 +40,7 @@ export interface FaceLivenessDetectorCoreProps {
    * Optional parameter for advanced options for the component
    */
   config?: FaceLivenessDetectorCoreConfig;
+  allowBackCamera?: boolean;
```

Comment:
I think we will want to have a conversion about whether this should be in the config or a top level prop.